test/integration/targets/win_psmodule/tasks/main.yml

```diff
@@ -87,6 +87,7 @@
       - "repo_fail|failed"
       - "module_check.stdout_lines[0] == '{{ powershell_module }}'"
       - "allow_clobber_check.stdout_lines[0] == '{{ allow_clobber_module }}'"
+  when: version.stdout_lines[0]|int >= 5
```

Comment:
Ok, all clear, thank you very mach for your help.
I' can use facts if they are disabled in test?

```yml
>>> Playbook: win_psmodule-azcq1C.yml
- hosts: windows
  gather_facts: False
  roles:
    - { role: win_psmodule }
```